qutebrowser/html/startpage.html

```diff
@@ -0,0 +1,177 @@
+{% extends "base.html" %}
+
+{% block style %}
+body {
+    background-color: #263238;
+    font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.wrapper {
+    width: 800px;
+    margin: 0 auto;
+}
+
+.header {
+    margin-top: 120px;
+    margin-bottom: 80px;
+}
+
+.title-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    max-width: 600px;
+    margin: auto;
+    padding: 15px;
+}
+
+.logo {
+    width: 105px;
+    height: 105px;
+    margin-bottom: 20px;
+}
+
+.title {
+    color: #c2e0ff;
+    font-size: 40px;
+}
+
+input {
+    font-size: 23px;
+    outline: none;
+    min-height: 50px;
+    width: 600px;
+    padding: 8px;
+    box-sizing: border-box;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background-color: #4e4f63db;
+    color: #cce1f7;
+}
+
+input::placeholder {
+    color: #bccbda;
+}
+
+.bookmarks {
+    display: flex;
+    flex-direction: column;
+}
+
+.bookmark-container {
+    border-color: #222d32;
+    background-color: #222d32;
+    border-radius: 8px;
+    border-width: 1px;
+    border-style: solid;
+    margin-bottom: 15px;
+    padding: 10px;
+}
+
+.bookmark-container-title {
+    margin: 0;
+    font-size: 25px;
+    font-family: monospace;
+    padding: 10px 10px 0 10px;
+    color: #84b4e6;
+}
+
+.link-container {
+    margin: 10px 15px;
+    font-size: 18px;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:link, a:visited {
+    color: #bdc4d4;
+}
+
+@media (prefers-color-scheme: light) {
+    body {
+        background-color: #ffffff;
```

Comment:
This seems a bit too "clinical" to me. What do you think? Maybe something like a light grey or bright blue-ish tone would work better?